src/SearchResults.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+function SearchResults(props) {
+  return (
+    <div>
+      <table className="table table-bordered">
+        <tr>
+          <th scope="col">Id</th>
+          <th scope="col">Title</th>
+          <th scope="col">First Name</th>
+          <th scope="col">Surname</th>
+          <th scope="col">Email</th>
+          <th scope="col">Room Id</th>
+          <th scope="col">Check in Date</th>
+          <th scope="col">Check out Date</th>
+        </tr>
+        {props.results.map(result => (
```

Comment:
Don't forget that when you iterate over an array, you'll need to make sure each element has a key. Right now there are errors throwing in the browser console because of this.
https://syllabus.codeyourfuture.io/react/week-1/lesson#keys